app/tests/spec/models/user.js

```diff
@@ -142,6 +142,75 @@ function (chai, sinon, p, Constants, Session, FxaClient, User) {
         });
     });
 
+
+    describe('in memory caching of signed in account', function () {
+      it('getSignedInAccount returns same instance from setSignedInAccount', function () {
+        var account = user.initAccount({ uid: 'uid', email: 'email'});
+        return user.setSignedInAccount(account)
+          .then(function () {
+            assert.equal(user.getSignedInAccount(), account);
+          });
+      });
+
+      it('getSignedInAccount returns same instance when called multiple times', function () {
+        return user.setSignedInAccount({ uid: 'uid', email: 'email'})
+          .then(function () {
+            assert.equal(user.getSignedInAccount(), user.getSignedInAccount());
+          });
+      });
+
+      it('getSignedInAccount returns different instance after clearSignedInAccount', function () {
+        var account = user.initAccount({ uid: 'uid', email: 'email'});
+        return user.setSignedInAccount(account)
+          .then(function () {
+            user.clearSignedInAccount();
+            assert.isFalse(user.getSignedInAccount() === account);
+          });
+      });
+
+      it('getSignedInAccount returns different instance after removeAccount', function () {
```

Comment:
The wording reflects the assertion being made in each of these tests (`assert.isFalse(user.getSignedInAccount() === account)`, i.e. the account it returns doesn't matter so long that it's not the previously cached account. Perhaps a better wording would be "getSignedInAccount does not return previously cached account after removeAccount".